contrib/mungegithub/blunderbuss.yml

```diff
@@ -0,0 +1,60 @@
+prefixMap:
+  test:
+    - ixdy
+
+  pkg/admission:
+    - lavalamp
+    - smarterclayton
+
+  pkg/api:
+    - bgrant0607
+
+  pkg/apiserver:
+    - lavalamp
+    - smarterclayton
+
+  pkg/authentication:
+    - liggitt
+    - erictune
+
+  pkg/client:
+    - lavalamp
```

Comment:
@smarterclayton just to be clear, this isn't a list of who will be Cc'd on PRs in each directory -- it's the pool of people the bot will randomly pick from to assign unassigned PRs that touch each directory